client/me/purchases/remove-purchase/index.jsx

```diff
@@ -257,7 +241,7 @@ const RemovePurchase = React.createClass( {
 			} ],
 			productName = getName( getPurchase( this.props ) );
 
-		if ( this.props.showChatLink && config.isEnabled( 'upgrades/precancellation-chat' ) ) {
+		if ( this.props.isHappychatAvailable && config.isEnabled( 'upgrades/precancellation-chat' ) ) {
```

Comment:
I can't get the "Need help?" button to trigger if I start my session at http://calypso.localhost:3000/me/purchases. However if I start my session at http://calypso.localhost:3000/me/chat and then navigate to purchases and try to cancel something, I see the button.

I'm guessing this page doesn't trigger Happychat connection, so `isHappychatAvailable` will never be `true` unless you first connect through another page. Ref: https://github.com/Automattic/wp-calypso/blob/master/client/me/help/help-contact/index.jsx#L69-L71